File: E-EstateV2 API/E-EstateV2 API/Models/FieldInfected.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace E_EstateV2_API.Models
{
    public class FieldInfected
    {
        [Key]
        public int Id { get; set; }
        public int areaInfected { get; set; }
        public DateTime dateScreening { get; set; }
        public DateTime? dateRecovered { get; set; }
        public string severityLevel { get; set; }
        public string remark { get; set; }
        public bool isActive { get; set; }
        public string createdBy { get; set; }
        public DateTime createdDate { get; set; }
        public string updatedBy { get; set; }
        public DateTime updatedDate { get; set; }

        [ForeignKey("FieldDiseaseId")]
        public int? fieldDiseaseId { get; set; }
        public FieldDisease FieldDisease { get; set; }

        [ForeignKey("FieldId")]
        public int? fieldId { get; set; }
        public Field Field { get; set; }
    }
}

File: E-EstateV2/src/app/_interface/fieldInfected.ts
export interface FieldInfected{
    id:number
    fieldDiseaseId:number
    fieldId:number
    fieldName:string
    area:number
    areaInfected:number
    dateScreening:Date
    createdBy: string
    createdDate: Date
    updatedBy: string
    updatedDate: Date
    remark:string
    isActive:boolean
    areaAffected:number
    severityLevel:string
    dateRecovered:Date
}